librz/core/cmd/cmd_analysis.c

```diff
@@ -430,6 +430,31 @@ static const char *help_msg_as[] = {
 	NULL
 };
 
+/**
+ * \brief Helper to get function in \p offset
+ *
+ * Case of overlapped functions is treated as an error.
+ */
+static RzAnalysisFunction *analysis_get_function_in(RzAnalysis *analysis, ut64 offset) {
+	RzAnalysisFunction *fcn = NULL;
+	RzList *list = rz_analysis_get_functions_in(analysis, offset);
```

Comment:
Irrelevant to this PR but maybe we want to return a vector from this function in the future, to reduce the memory consumption.

Reply:
Ah, ok, makes sense then.